bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please revert this file, it is no longer used in 4.0.

Reply:
No, I need it for downgrade compability ->[ see here ](https://community.openhab.org/t/binding-compatibilty/145185/7)

Reply:
You need to do that in a separate (local) branch. This PR is targeting **main** which is 4.0. Only bugfixes are backported to 3.4.x, and they will usually be cherry-picked branches merged into main.

Reply:
ok, will do as the last change, if everything else is fine.